--- SuiteSparse_config.mk
+++ SuiteSparse_config.mk
@@ -158,7 +158,7 @@
             #   $(MKLROOT)/lib/intel64/libmkl_intel_thread.a \
             #   -Wl,--end-group -lpthread -lm
             # using dynamic linking:
-            BLAS = -lmkl_intel_lp64 -lmkl_core -lmkl_intel_thread -lpthread -lm
+            BLAS = -lmkl_intel_lp64 -lmkl_tbb_thread -lmkl_core -ltbb -lstdc++ -lpthread -lm -ldl
             LAPACK =
         else
             # use the OpenBLAS at http://www.openblas.net
